typescript-webpack-sample/src/types.d.ts

```diff
@@ -0,0 +1,3 @@
+// Basic declarations as `any` until proper UXP types exist:
```

Comment:
Maybe it's my lack of TypeScript knowledge but I don't really understand the point of this file. I don't see it used anywhere

Reply:
The thing is that TypeScript would create errors since it can't find type definitions for these modules. This is why I have to declare them as "existing" (of type `any`) to call various functions from the `uxp` module in https://github.com/AdobeDocs/uxp-photoshop-plugin-samples/pull/66/files#diff-6e658eca5b5fb3819cfcb6c705898990006444f84c8300feba1bed5eb7d67ce1R13.